docs/admin/templates/extending-templates/workspace-tags.md

```diff
@@ -66,6 +70,14 @@ the workspace owner to change a provisioner group (due to different tags). In
 most cases, `coder_parameter`s backing `coder_workspace_tags` should be marked
 as immutable and set only once, during workspace creation.
 
+**We recommend only using the following as inputs for `coder_workspace_tags`:**
+
+- Static values (e.g. `"developers"`),
```

Comment:
```suggestion
- Static values (e.g. `developers`),
```

to match <https://coder.com/docs/admin/templates/extending-templates/workspace-tags#dynamic-tag-selection>

Reply:
Resolved with alternative suggestion!